unittests/tools/test_checkmarx_parser.py

```diff
@@ -7,56 +7,58 @@
 from ..dojo_test_case import DojoTestCase, get_unit_tests_path
 
 
+def init(report_filename):
+    my_file_handle = open(report_filename)
+    product = Product()
+    engagement = Engagement()
+    test = Test()
+    engagement.product = product
+    test.engagement = engagement
+    return my_file_handle, product, engagement, test
+
+
+def teardown(my_file_handle):
+    my_file_handle.close()
+
+
```

Comment:
What is the benefit of moving these outside of the class?

Reply:
My IDE suggested converting the methods to functions and I accepted it without thinking twice. I have now revoked the change.